### PR TITLE
bug fixing

### PR DESCRIPTION
**Pull request by meisen45**

### CODE REVIEW

File: app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java [hunk 1]
```diff
@@ -91,13 +93,21 @@
     private ListAdapter<ListRow> mRowsAdapter;
     private VideoEntity mSelectedVideo;
     private VideosViewModel mViewModel;
+    private LiveDataRowPresenter mRowPresenter;
```
Comment (dakegu2) [inline app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java]: doesn't need persist as a member

Comment (meisen45) [reply app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java]: done

File: app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java [hunk 2]
```diff
@@ -96,8 +98,15 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        // presenter for each row
+        LiveDataRowPresenter mRowPresenter = new LiveDataRowPresenter();
```
Comment (dakegu2) [inline app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java]: mRowPresenter -> rowPresenter

Comment (meisen45) [reply app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/room/LiveDataFragment.java]: done, thank you for pointing it out

**Pull request merged**